src/annmas/annotate/command.py

```diff
@@ -137,7 +147,10 @@ def main(model, threads, output_bam, input_bam):
         out_header = pysam.AlignmentHeader.from_dict(out_bam_header_dict)
 
         # Start output worker:
-        output_worker = mp.Process(target=_write_thread_fn, args=(results, out_header, output_bam, pbar))
+        res = manager.dict({"num_reads_annotated": 0, "num_sections": 0})
```

Comment:
Enables us to move the logging statement to the outer calling function, which again improves the log output formatting.